### DISCUSSION TITLE
现在taro项目在微信小程序下还需要关闭es6吗

### DISCUSSION THREAD

**Discussion by kuonji-arisu** [NervJS/taro#13889] [Q & A]
我看文档上有说需要在开发者工具里关闭es5选项
但是我使用官方的vue3模板在开发者工具真机调试的时候，会提示es6错误而终止调试

**Comment by TheKonka** [comment]
![image](https://github.com/NervJS/taro/assets/22173084/506cff02-75c1-4711-b052-66e518e7da3e)
这个吗？我一直都是勾选☑️的，好像没有问题，暂时不知道 为什么文档里面说要关闭
